app/lib/hyrax/hirmeos/metrics_tracker.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+require "hyrax/hirmeos/client"
+require "hyrax/hirmeos/work_factory"
```

Comment:
Since these were moved to `app` are they autoloaded and the requires can be dropped?